### PR TITLE
Various minor fixes

### PR DESCRIPTION
**Pull request by dwightguth**
**please review**

### CODE REVIEW

**Comment by dwightguth** [comment]
@chathhorn please review

File: semantics/language/execution/expr/function-call.k [hunk 1]
```diff
@@ -210,6 +210,8 @@ module C-EXPR-FUNCTION-CALL
           requires fromConstantExpr(T)
      rule _:KResult := tv(_:CValue, T:Type => stripConstants(T))
           requires fromConstantExpr(T)
+     rule _:KResult := te(_, T:Type => stripConstants(T))
+          requires fromConstantExpr(T)
```
Comment (chathhorn) [inline semantics/language/execution/expr/function-call.k]: I don't think this should be necessary -- te() => tv() in the execution semantics (see init.k:58).

File: semantics/language/translation/typing/canonicalization.k [hunk 2]
```diff
@@ -275,6 +275,7 @@ module C-TYPING-CANONICALIZATION
           <structs> Structs:Map </structs>
           <curr-scope> blockScope(_, BlockNum:Int) </curr-scope>
           <curr-tu> Tu:String </curr-tu>
+          <block-history> ListItem(BlockNum:Int) ...</block-history>
```
Comment (chathhorn) [inline semantics/language/translation/typing/canonicalization.k]: Necessary? I suspect it got added during the merge.

**Comment by chathhorn** [comment]
Looks good!

**Pull request merged**